samples/apps/copilot-chat-app/webapi/appsettings.json

```diff
@@ -180,7 +184,7 @@
   "Logging": {
     "LogLevel": {
       "Default": "Warning",
-      "SemanticKernel.Service": "Information",
+      "SemanticKernel.Service": "Debug",
```

Comment:
Did you mean to check this line in? Sounds like we could be in for lots of logging now...

Reply:
reverting